### ISSUE TITLE
Support for Cucumber 4.0.0

### ISSUE THREAD

**Issue by twometresteve** [burtlo/yard-cucumber#100]
Cucumber 4.0.0 was released last week (after about 2 years of development).

Do you plan to support it?  Is it likely to be much work?

TIA,
Steve.

**Comment by Smarre** [comment]
Maybe someone needs to fork this gem...

**Comment by twometresteve** [comment]
> Maybe someone needs to fork this gem...

Yes, that's what we did in the end - as had many others it seems.

**Comment by Smarre** [comment]
What is the most popular/maintained fork?

**Comment by twometresteve** [comment]
No idea about popularity, I'm afraid.  I created https://github.com/bugsnag/yard-cucumber recently and made minimal changes that allowed us to update our tooling to Cucumber v7 - but we're not planning to actively develop the repo further (in fact, I would not consider it maintained in that sense).

**Comment by pgundlupetvenkatesh** [comment]
@twometresteve Thanks for creating [bugsnag/yard-cucumber](https://github.com/bugsnag/yard-cucumber) repo it's been helpful for our team at work. I know you have made it clear you won't be maintaining/developing that repo further. As Cucumber v8.0.0 is out there do you mind adding support for that please?

**Comment by twometresteve** [comment]
@pgundlupetvenkatesh - PRs welcome on that repo.
